apps.go

```diff
@@ -94,20 +94,21 @@ type App struct {
 		Username string `json:"username"`
 		Email    string `json:"email"`
 	} `json:"owner"`
-	GitUrl         string          `json:"git_url"`
-	Url            string          `json:"url"`
-	BaseURL        string          `json:"base_url"`
-	Status         AppStatus       `json:"status"`
-	LastDeployedAt *time.Time      `json:"last_deployed_at"`
-	LastDeployedBy string          `json:"last_deployed_by"`
-	CreatedAt      *time.Time      `json:"created_at"`
-	UpdatedAt      *time.Time      `json:"update_at"`
-	Links          *AppLinks       `json:"links"`
-	StackID        string          `json:"stack_id"`
-	StickySession  bool            `json:"sticky_session"`
-	ForceHTTPS     bool            `json:"force_https"`
-	RouterLogs     bool            `json:"router_logs"`
-	Flags          map[string]bool `json:"flags"`
+	GitUrl         string           `json:"git_url"`
+	Url            string           `json:"url"`
+	BaseURL        string           `json:"base_url"`
+	Status         AppStatus        `json:"status"`
+	LastDeployedAt *time.Time       `json:"last_deployed_at"`
+	LastDeployedBy string           `json:"last_deployed_by"`
+	CreatedAt      *time.Time       `json:"created_at"`
+	UpdatedAt      *time.Time       `json:"update_at"`
+	Links          *AppLinks        `json:"links"`
+	StackID        string           `json:"stack_id"`
+	StickySession  bool             `json:"sticky_session"`
+	ForceHTTPS     bool             `json:"force_https"`
+	RouterLogs     bool             `json:"router_logs"`
+	Flags          map[string]bool  `json:"flags"`
+	Limits         map[string]int64 `json:"limits"`
```

Comment:
My question is: are we sure it's an int, can't it be a float64 sometimes ?

Reply:
Currently everything is in integer, but indeed if we add a field of another type it could cause some problems. I can put an interface instead if you want

Reply:
I'm afraid it's not safe over time actually

Reply:
(So maybe `interface{}` is safer, like that at each usage, the developer has to wonder, which type it is)

Reply:
Ok i change it to `map[string]interface{}`

Reply:
I prefer the `int64` type. It has been safe for the past 5 years, shouldn't we change this to `interface{}` when it becomes useful?

Reply:
Shouldn't we directly put a struct here ?
Because if we add a non int64 field here we'll break applications when changing this.
Adding a struct would let us maintaining this without breaking anything ?

Reply:
Hmm, the question is, will we think about this when we add another type?

Reply:
I don't see how we would break applications, knowing that the field is not already present

Reply:
Then the question is at a different level: I don't think this lib should adapt to the organizational problem of how to think about updating our libs when we update the output of the API. That's why we have this PR template on the API: https://github.com/Scalingo/api/blame/master/.github/pull_request_template.md#L4

Reply:
To me there is also the thing that we should create libs with nice API that we don't break for any business reason, that's why I was in favor of `interface{}` since there are unknowns here and it would prevent us from breaking the API in the future

Reply:
I'm in favor of the `interface{}`

Reply:
Let's go then